src/components/userdetail/UserDetail.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import Media from '~/components/media/Media';
+import Button from '~/components/button/Button';
+import { Image } from '~/assets/image';
+import { FAVORITE_ICON } from '~/components/icons/icon';
+
+const UserDetail = () => {
+	return (
+		<div className="user-detail">
```

Comment:
@Esha-Rajbhandari Here we have some indentation problems

Reply:
🆗